### PR TITLE
Istio 1.4.800 - 2.5

### PR DESCRIPTION
**Pull request by brendarearden**
https://github.com/rancher/rancher/issues/26784

### CODE REVIEW

**Review by daxmc99 — DISMISSED** [review]

File: charts/rancher-istio/1.4.800/questions.yaml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 labels:
-  rancher.istio.v1.4.700: 1.4.7
+  rancher.istio.v1.4.800: 1.4.8
 rancher_min_version: 2.3.4-rc1
```
Comment (cbron) [inline charts/rancher-istio/1.4.800/questions.yaml:3]: Need to ask if we should also deprecate the 1.4.700 chart, think the answer is no.

**Review by cbron — COMMENTED** [review]
Just a question but looks good. After this is merged, need to also PR to dev-v2.3 and dev-v2.4 branches.

File: charts/rancher-istio/1.4.800/questions.yaml (hunk 1 above)
Comment (brendarearden) [reply charts/rancher-istio/1.4.800/questions.yaml:3]: per Denise we will be deprecating, so I will update the pr

File: charts/rancher-istio/1.4.700/questions.yaml [hunk 2]
```diff
@@ -1,3 +1,4 @@
 labels:
   rancher.istio.v1.4.700: 1.4.7
 rancher_min_version: 2.3.4-rc1
+rancher_max_version: 2.3.4-rc2
```
Comment (cbron) [inline charts/rancher-istio/1.4.700/questions.yaml]: You can keep this as rc1

File: charts/rancher-istio/1.4.800/questions.yaml [hunk 3]
```diff
@@ -1,3 +1,3 @@
 labels:
   rancher.istio.v1.4.800: 1.4.8
-rancher_min_version: 2.3.4-rc1
+rancher_min_version: 2.3.4-rc2
```
Comment (cbron) [inline charts/rancher-istio/1.4.800/questions.yaml]: And set min to rc1 here

**Review by cbron — CHANGES_REQUESTED** [review]
Small change, doesn't really matter but want to keep simple as not to be confusing in future for others.

**Review by cbron — APPROVED** [review]
lgtm

**Pull request merged**